mmdeploy/codebase/mmseg/models/decode_heads/ema_head.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn.functional as F
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmseg.models.decode_heads.ema_head.EMAModule.forward')
+def ema_module__forward(ctx, self, feats):
+    """Rewrite `forward` for default backend.
+
+    Replace torch.einsum with other operations.
```

Comment:
Shall we just rewrite torch.einsum rather than mmseg codes?

Reply:
`torch.einsum` is a bit complex when considering all cases. Could be handled in another PR.